pom.xml

```diff
@@ -16,20 +16,10 @@
 
 	<properties>
 		<java.version>17</java.version>
+		<!-- If you want to use a different version of Testcontainers -->
+		<testcontainers.version>1.19.1</testcontainers.version>
 	</properties>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.testcontainers</groupId>
-                <artifactId>testcontainers-bom</artifactId>
```

Comment:
Did we get rid of the bom?

Reply:
Is this Spring-managed now?

Reply:
Yes, from Spring Boot 3.1.0 they have configured testcontainers-bom in parent pom.